### ISSUE TITLE
WCAG AAA contrast failures (needs design)

### ISSUE THREAD

**Issue by brdunfield** [cds-snc/platform-forms-client#324]
If we are only aiming for AA then this issue can be closed / ignored/

Error text on widgets ("Please complete the required field to continue") Fails WCAG 2.1 for regular text for AAA with a 6.5:1 ratio. Passes at AA level.

https://images.zenhubusercontent.com/60d4bcc1cb1e877f2aba6493/48a966b3-8109-48cc-b99e-67257e341447

Submit button in its default state fails contrast at WCAG 2.1 at AAA for regular text but passes for large text (I think this counts as regular text based on the size of the font). Passes at AA level.

https://images.zenhubusercontent.com/60d4bcc1cb1e877f2aba6493/e34e25ef-1349-4392-a2cc-3cdd8e610071

**Comment by timarney** [comment]
Closing as a stale issue  --- styling is up to date with the design system now.

@brdunfield 👋
